content/20.treatments.md

```diff
@@ -805,6 +805,17 @@ A list of current studies and their results, using carefully the information req
 **Summary.**
 Summarize the state of the neutralizing antibody approach.
 -->
+
+##### Adjuvants for Vaccines
+
+Adjuvants are sometimes included within vaccines, especially vaccines other than live attenuated and inactivated viruses, in order to enhance the immune response.
+A review on the development of SARS-CoV-2 vaccines [@doi:10.3390/vaccines8020153] also included a brief summary of considerations for adjuvants to research for the vaccines including a brief description of some already commonly used adjuvants.
+Different adjuvants can regulate different types of immune responses, so the type of adjuvant or combination of adjuvants for use in a vaccine depends on the type of vaccine and the research into what works with respect to efficacy and safety.
+Adjuvants include a variety of molecules or larger microbial-related products which sometimes are or include immunostimulants or immunomodulators or more generally have an effect on the immune system or immune responses of interest.
+A variety of possible mechanisms for adjuvants have been researched [@doi:10.1038/nri2510; @doi:10.3389/fimmu.2018.02619; @doi:10.3390/vaccines8010128], including induction of DAMPs (damage-associated molecular patterns) which can be recognized by certain pattern recognition receptors (PRRs) of the innate immune system, being or including or functioning as PAMPs (pathogen-associated molecular patterns) which can also be recognized by certain pattern recognition receptors (PRRs), and more generally enhancing humoral (type 2) or cellular (type 1) immune responses.
```

Comment:
```suggestion
A variety of possible mechanisms for adjuvants have been researched [@doi:10.1038/nri2510; @doi:10.3389/fimmu.2018.02619; @doi:10.3390/vaccines8010128].
One possibility is the induction of DAMPs (damage-associated molecular patterns) that can be recognized by certain pattern recognition receptors (PRRs) of the innate immune system.
Another is that they can either be, include, or function as PAMPs (pathogen-associated molecular patterns) that can also be recognized by certain PRRs.
In general terms, adjuvants enhance the humoral (type 2) or cellular (type 1) immune responses.
```

Reply:
I'll take the suggestion to separate it more obvious but keeping it together by using ":" & ";"